src/bundle/eZ/RichText/Renderer.php

```diff
@@ -127,7 +127,7 @@ public function renderContentEmbed($contentId, $viewType, array $parameters, $is
             /** @var \eZ\Publish\API\Repository\Values\Content\Content $content */
             $content = $this->repository->sudo(
                 function (Repository $repository) use ($contentId) {
-                    return $repository->getContentService()->loadContent($contentId);
+                    return $repository->getContentService()->loadContent((int)$contentId);
```

Comment:
TBH you should fix it [here](https://github.com/ezsystems/ezplatform-richtext/blob/v2.0.0-beta4/src/lib/eZ/RichText/Converter/Render/Embed.php#L89) and enforce strict types in `renderContentEmbed`.

But can be done as a follow-up when handling `renderContentEmbed` by introducing strict types there and removing type cast here.